exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryResources.java

```diff
@@ -96,10 +95,12 @@ public QueryResult submitQueryJSON(QueryWrapper query) throws Exception {
   @Produces(MediaType.TEXT_HTML)
   public Viewable submitQuery(@FormParam("query") String query,
                               @FormParam("queryType") String queryType,
-                              @FormParam("autoLimit") String autoLimit) throws Exception {
+                              @FormParam("autoLimit") String autoLimit,
+                              @FormParam("defaultSchema") String defaultSchema) throws Exception {
```

Comment:
Nice improvement! I wonder, would it make sense to accept a list of name/value pairs so we can also pass along session options? We could identify the schema as a REST-specific property in that list.